xbmc/interfaces/legacy/ModuleXbmcvfs.h

```diff
@@ -278,10 +278,13 @@ namespace XBMCAddon
 #ifdef DOXYGEN_SHOULD_USE_THIS
     ///
     /// \ingroup python_xbmcvfs
-    /// @brief \python_func{ xbmcvfs.rmdir(path) }
+    /// @brief \python_func{ xbmcvfs.rmdir(path, [force]) }
     /// Remove a folder.
     ///
-    /// @param path                  Folder to remove
+    /// @param path                  string - Folder to remove
+    /// @param force                 [opt] bool - Force directory removal
+    ///                              (default False). This can be useful
+    ///                              if the directory is not empty.
     /// @return                      True if successed
```

Comment:
Your changes are good, is it worthwhile just altering the grammar on this line as well while you are at it?

```
/// @return       True if succeeded
```

or alternatively successful would work as weill

Reply:
Sure, another shitstorm coming to Jenkins in 3,2,1... :)